ModuleNotFoundError: No module named 'typing_extensions'

🐞 **Describe the bug**
<!-- A clear and concise description of what the bug is, on the next line. -->

After updating aiohttp from 3.6.2 to 3.7.0, the [gql](https://github.com/graphql-python/gql) tests are failing on python 3.8 with the Exception `ModuleNotFoundError: No module named 'typing_extensions'`

```
tests/conftest.py:15: in <module>
    from aiohttp.test_utils import TestServer as AIOHTTPTestServer
.tox/py38/lib/python3.8/site-packages/aiohttp/test_utils.py:43: in <module>
    from .web import (
.tox/py38/lib/python3.8/site-packages/aiohttp/web.py:23: in <module>
    from .web_app import Application as Application
.tox/py38/lib/python3.8/site-packages/aiohttp/web_app.py:39: in <module>
    from .web_middlewares import _fix_request_current_app
.tox/py38/lib/python3.8/site-packages/aiohttp/web_middlewares.py:7: in <module>
    from .web_urldispatcher import SystemRoute
.tox/py38/lib/python3.8/site-packages/aiohttp/web_urldispatcher.py:36: in <module>
    from typing_extensions import TypedDict
E   ModuleNotFoundError: No module named 'typing_extensions'

```

💡 **To Reproduce**

Check the PR [#156](https://github.com/graphql-python/gql/pull/156)

--- a/CHANGES/5107.bugfix
+++ b/CHANGES/5107.bugfix
@@ -1,0 +1,1 @@
+Always require ``typing_extensions`` library.

--- a/setup.py
+++ b/setup.py
@@ -71,7 +71,7 @@ install_requires = [
     "async_timeout>=3.0,<4.0",
     "yarl>=1.0,<2.0",
     'idna-ssl>=1.0; python_version<"3.7"',
-    'typing_extensions>=3.6.5; python_version<"3.7"',
+    "typing_extensions>=3.6.5",
 ]
 
 

--- a/tests/test_web_functional.py
+++ b/tests/test_web_functional.py
@@ -1952,7 +1952,8 @@ async def test_read_bufsize(aiohttp_client) -> None:
 
 
 @pytest.mark.parametrize(
-    "status", [101, 204],
+    "status",
+    [101, 204],
 )
 async def test_response_101_204_no_content_length_http11(
     status, aiohttp_client

